### PR TITLE
Assessment tool deck - First draft

### PR DESCRIPTION
**Pull request by smellems**
https://sara-sabr.github.io/util-presentation/presentation.html?url=https://raw.githubusercontent.com/sara-sabr/ITStrategy/assesstooldeck/presentations/en/Assessment-tool.md

### CODE REVIEW

**Comment by ElminaIusifova** [comment]
My comments:
**Slide 3: DEVOPS RESEARCH & ASSESSMENT**:
It is too wordy, we can mention this verbally but leave minimum content on the slide
**Slide 4: CAPABILITIES:**
We can visualize this slide in a diagram (on my to-do list)
**Slide 5 HOW TO TRANSFORM**
We can do our own infographic on how to transform or take the image  from here https://cloud.google.com/solutions/devops/devops-culture-transform "The four steps of the improvement" (tbd)
Steps slides can be visualized  and we can also add the assessment tool process flow what I included on the landing page
Let's discuss it at our meeting

**Review by ElminaIusifova — COMMENTED** [review]
Commented provided below. Some slides need to be visualized

**Comment by smellems** [comment]
Related to #1219

**Review by ElminaIusifova — APPROVED** [review]
